melosys-eessi-test/pom.xml

```diff
@@ -45,6 +45,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Hvor blir denne brukt?

Reply:
For å eksponere en egendefinert `Unleash` (`FakeUnleash`) i test for å toggle feature av og på 🙂